src/pageload.js

```diff
@@ -3,7 +3,7 @@ import firstImage from './assets/pexels-dhyamis-kleber-3584317.jpg';
 import fifthImage from './assets/pexels-tookapic-7782.jpg';
 import thirdImage from './assets/pexels-chevanon-photography-312418.jpg';
 
-export function pageload() {
+export let pageload =  () => {
```

Comment:
- [ ] All arrow functions should be declared using `const` not `let` as they are not going to be reassigned, kindly modify them before merging.